projects/plugins/jetpack/modules/theme-tools/compat/twentytwenty.php

```diff
@@ -124,7 +125,7 @@ function twentytwenty_infinity_accent_color_css() {
 	}
 	',
 		$color_info['content']['accent'],
-		$color_info['content']['background'],
+		$color_info['content']['background'] ?? '#fff',
```

Comment:
Themes always set the other two, just not this one?

Reply:
Oh, duh, "projects/plugins/jetpack/modules/theme-tools/compat/**twentytwenty.php**" :sweat_smile: